examples/basic_example.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";

```

Comment:
put this into examples/basic_example/src, so that each example project stands on it's own.